### PR TITLE
[WebProfilerbundle] add status code to search form

### PR DESCRIPTION
**Pull request by oktapodia**

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/Profiler/BaseMemcacheProfilerStorage.php [hunk 1]
```diff
@@ -63,11 +63,11 @@ public function find($ip, $url, $limit, $method, $start = null, $end = null)
 
             $values = explode("\t", $item, 7);
             list($itemToken, $itemIp, $itemMethod, $itemUrl, $itemTime, $itemParent) = $values;
-            $statusCode = isset($values[6]) ? $values[6] : null;
+            $itemStatusCode = isset($values[6]) ? (int) $values[6] : null;
```
Comment (AbdoulNdiaye) [inline src/Symfony/Component/HttpKernel/Profiler/BaseMemcacheProfilerStorage.php]: This is not an item. You should keep the name "statusCode"

File: src/Symfony/Component/HttpKernel/Profiler/MysqlProfilerStorage.php [hunk 2]
```diff
@@ -64,6 +64,11 @@ protected function buildCriteria($ip, $url, $start, $end, $limit, $method)
             $args[':method'] = $method;
         }
 
+        if ($statusCode) {
+            $criteria[] = 'status_code = :status_code';
```
Comment (AbdoulNdiaye) [inline src/Symfony/Component/HttpKernel/Profiler/MysqlProfilerStorage.php]:
```php
        if ($statusCode) {
            $criteria[]           = 'status_code = :status_code';
            $args[':status_code'] = $statusCode;
        }
```

File: src/Symfony/Component/HttpKernel/Profiler/PdoProfilerStorage.php [hunk 3]
```diff
@@ -127,16 +127,17 @@ public function purge()
...
-     * @param string $end    The end period to search to
-     * @param string $limit  The maximum number of tokens to return
-     * @param string $method The request method
+     * @param string $ip         The IP
+     * @param string $url        The URL
+     * @param string $start      The start period to search from
+     * @param string $end        The end period to search to
+     * @param string $limit      The maximum number of tokens to return
+     * @param string $method     The request method
+     * @param int    $statusCode The request method
```
Comment (AbdoulNdiaye) [inline src/Symfony/Component/HttpKernel/Profiler/PdoProfilerStorage.php]: The comment is false. Change the description

File: src/Symfony/Component/HttpKernel/Profiler/BaseMemcacheProfilerStorage.php (hunk 1 above)
Comment (oktapodia) [inline src/Symfony/Component/HttpKernel/Profiler/BaseMemcacheProfilerStorage.php]: The name statusCode already exist :/

File: src/Symfony/Component/HttpKernel/Profiler/MysqlProfilerStorage.php (hunk 2 above)
Comment (oktapodia) [inline src/Symfony/Component/HttpKernel/Profiler/MysqlProfilerStorage.php]: :+1:

**Pull request closed without merging**